SYCL/KernelAndProgram/undefined-symbol.cpp

```diff
@@ -1,3 +1,8 @@
+// because of problems getting GPU driver uplifted on GitHub Actions
+// this test both fails on GHA and passes on Jenkins. 
+// Disabling it until this is sorted
+// UNSUPPORTED: windows || linux
```

Comment:
Probably it would be better to use // REQUIRES: TEMPORARILY_DISABLED, so that it is more clear that this test is temporarily disabled.

Reply:
Is that how we normally do it? That's a good BKM.

Reply:
Yes. It also allows to grep for  TEMPORARILY_DISABLED and see our debts.